src/cmd-buildextend-se

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+dn=$(dirname "$0")
+# shellcheck source=src/cmdlib.sh
+. "${dn}"/cmdlib.sh
+
+
+image_type=se
+
+print_help() {
+    cat 1>&2 <<EOF
+Usage: coreos-assembler buildextend-${image_type} --help
+       coreos-assembler buildextend-${image_type} [--build ID]
+
+  Build a bare metal image.
+EOF
+}
+
+# Parse options
+rc=0
+build=
+options=$(getopt --options h --longoptions help,build,hostkey: -- "$@") || rc=$?
+[ $rc -eq 0 ] || {
+    print_help
+    exit 1
+}
+eval set -- "$options"
+while true; do
+    case "$1" in
+        -h | --help)
+            print_help
+            exit 0
+            ;;
+        --build)
+            build=$2
+            shift
+            ;;
+        --hostkey)
+            hostkey=$HOME/hostkeys/$2
```

Comment:
I think hostkey here should be directly derived from the argument, and not be expected to be under `$HOME/hostkeys/`, as that directory is not guaranteed to be mounted inside the cosa container.
```suggestion
            hostkey=$2
```

Reply:
this is for`cosa` command, which should have smth like ` -v ${HOME}/prod-keystore-master:/home/builder/hostkeys/:ro \` to map hostkey (universal one) into running container. so this `hostkeys` folder is used just on my builder for now

Reply:
Wouldn't it be easier to have it in the already mounted `/srv` folder? Instead of mounting a new folder into the container + having a hardcoded Path?